extension/healthcheckextension/config.go

```diff
@@ -23,7 +23,7 @@ import (
 type Config struct {
 	configmodels.ExtensionSettings `mapstructure:",squash"`
 
-	// Port is the port used to publish the health check status.
-	// The default value is 13133.
-	Port uint16 `mapstructure:"port"`
+	// Endpoint is the address and port used to publish the health check status.
```

Comment:
Can we do this change in two stages? Add the new one and deprecate the old option in one step, remove the deprecated in the next step. This is the type of change that doesn't need to be breaking for current users.

Reply:
Good point, I'll bring it back next commit